app/deed/search/server.py

```diff
@@ -0,0 +1,6 @@
+from flask import redirect
+
+def register_routes(blueprint):
+    @blueprint.route('/deed/search', methods=['POST'])
+    def redirect_to_view_deed():
+        return redirect("/deed/view", code=303)
```

Comment:
Can you use the `flask.url_for` function rather than hard coding the url. Just decreases the fragility of the code. They are namespaced by the name you gave the blueprint and the name of the function, so `view.view_deed`.